doc/errors.org

```diff
@@ -0,0 +1,95 @@
+* Errors
+https://ziglang.org/documentation/0.10.1/#Errors
+In ZIG errors are juste values.
+Those values are those of an special type of enum "error".
+When you declare your enum using this keyword instead of "enum", the compiler is going to know that those values are going to be errors, therefore they can be catched, tried, ...
+
+
+** Handling errors in the application flow
+Functions can return either a value or an error, this is done by adding the "!" to the return type of a function
+With the "!" the return type of the function is the coerced "anyerror!u64"
+
+When a function can return an error we have to either "try" or "catch" it.
+
+#+source: simple
+#+begin_src zig :imports '(std)
+fn maybeErrorMaybeValue(isError: bool) !u8 {
+    // ... might return an u8 or an error
+    if (isError) {
+        return error.Bar;
+    } else {
+        return 2;
+    }
+}
+
+test "return value or error" {
+    // "try" version
+    const res = try maybeErrorMaybeValue(false); // if error, return the error
+    try std.testing.expect(res == 2);
+    
+    // "catch" version
+    const err = maybeErrorMaybeValue(true) catch 3;
+    try std.testing.expect(err == 3);
+}
+
+test "should panic" {
+    _ = maybeErrorMaybeValue(true) catch unreachable;
+}
+#+end_src
+
+** Errdefer
+This exemple is taken from: https://ziglang.org/documentation/0.10.1/#errdefer
```

Comment:
Doc is for 0.10

Reply:
I accidently was reading the 0.10 doc while doing this, I did all my exemples on 0.11 and just checked the 2 docs and see no difference between the 2 on this subject

Reply:
Ok, mabe we can change the link to 0.11